src/main/java/io/stargate/sgv2/jsonapi/service/FeatureFlags.java

```diff
@@ -0,0 +1,7 @@
+package io.stargate.sgv2.jsonapi.service;
+
+/** Hack / temp class to have run time checking for if Tables are supported */
+public final class FeatureFlags {
+
+  public static final boolean TABLES_SUPPORTED = Boolean.getBoolean("stargate.tables.supported");
```

Comment:
Note for other reviewers: this is changed to Q property, renamed, in a later PR.

Reply:
Thanks! 
I see this is the feature flag we keep talk about.